scripts/ci/lib.sh

```diff
@@ -152,6 +152,28 @@ get_central_debug_dump() {
 
     roxctl -e "${API_ENDPOINT}" -p "${ROX_PASSWORD}" --insecure-skip-tls-verify central debug dump --output-dir "${output_dir}"
     ls -l "${output_dir}"
+
+    local metrics_output
+    local csv_output
+    metrics_output="$(mktemp --suffix=.prom)"
+    csv_output="$(mktemp --suffix=.csv)"
+    unzip -p "${output_dir}/*.zip" metrics-1 > "${metrics_output}"
+
+    # TODO(janisz): This is also declared in stackrox/tests/e2e/run-scale.sh:77-80
+    # We should move it into tools modules to be not be surprised by using the latest version
+    # and setup dependabot to automatically bump it.
+    go install github.com/stackrox/prometheus-metric-parser@latest
+
+    prometheus-metric-parser single --file /tmp/metrics \
+      | awk -F"," \
+      -v build_tag="${STACKROX_BUILD_TAG}" \
+      -v build_id="${BUILD_ID}" \
+      -v orchestrator="${ORCHESTRATOR_FLAVOR}" \
+      -v job_name="${JOB_NAME:-missing}" \
+      'BEGIN { OFS = "," } {$4=build_tag;$5=build_id;$6=orchestrator;$7=job_name; print}' > "${csv_output}"
```

Comment:
I can see the test-metrics .csv files have a header line with comma-separated column names.
I might be missing something, but it seems `--skip_leading_rows=1` in  `bq` invocation is expecting such header...

As an aside, if the oder of columns here is supposed to match some order specified elsewhere, a link would be of great help to future maintainers.